tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -137,11 +134,8 @@ describe('Performing an upgrade', () => {
   });
 
   it('should display upgrade page even without upgrade logs', async () => {
-    await loginPage.cookieLogin({
-      username: constants.USERNAME,
-      password: constants.PASSWORD,
-      createUser: false
-    });
+    await loginPage.login({ username: constants.USERNAME, password: constants.PASSWORD, loadPage: false });
+    await oldNavigationPage.goToBase();
```

Comment:
@kapelo you don't need to change this here because, at this point, the new version will be already installed.

This test is to verify that we can upgrade from an old version to the new one that we are releasing. That is why in the `before` method, we needed to change the login because, at the beginning of the test, we are on an old version, but in this test, the version is already updated